src/notebooks/controllers/commands/installPythonControllerCommands.ts

```diff
@@ -162,11 +162,6 @@ export class InstallPythonControllerCommands implements IExtensionSingleActivati
         if (!this.extensionChecker.isPythonExtensionInstalled) {
             sendTelemetryEvent(Telemetry.PythonExtensionNotInstalled, undefined, { action: 'displayed' });
 
-            if (!(await this.shouldInstallExtensionPrompt())) {
```

Comment:
No point prompting users to install an extension, VS Code already does this.
If we leave this, then users will get two prompts to install an extension.